src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -1,5 +1,5 @@
 // ***********************************************************************
-// Copyright (c) 2011-2014 Charlie Poole, Rob Prouse
+// Copyright (c) 2011�2019 Charlie Poole, Rob Prouse
```

Comment:
```suggestion
// Copyright (c) 2011-2019 Charlie Poole, Rob Prouse
```

Whichever type of dash this is, it borks in my VS diff tool! 😂 Swap for a normal hyphen?

Reply:
I've been using the n-dash (preferred for ranges) other places in the NUnit org, but I could swap it. The problem is that the file was saved with Western European codepage encoding instead of UTF8. Is it alright if I fix the problem by saving it as UTF8?